pom.xml

```diff
@@ -33,107 +33,8 @@
-    <module>osgp/protocol-adapter-iec60870/osgp-protocol-simulator-iec60870</module>
-
-    <!-- protocol-adapter-mqtt -->
-    <module>osgp/protocol-adapter-mqtt/parent-pa-mqtt</module>
-    <module>osgp/protocol-adapter-mqtt/osgp-protocol-adapter-mqtt</module>
-    <module>osgp/protocol-adapter-mqtt/osgp-protocol-simulator-mqtt</module>
-
-    <!-- platform -->
-    <module>osgp/platform/parent-platform</module>
-    <module>osgp/platform/osgp-domain-core</module>
-    <module>osgp/platform/osgp-domain-logging</module>
-    <module>osgp/platform/osgp-domain-microgrids</module>
-    <module>osgp/platform/osgp-domain-distributionautomation</module>
-
-    <module>osgp/platform/osgp-adapter-domain-shared</module>
-
-    <module>osgp/platform/osgp-adapter-domain-admin</module>
-    <module>osgp/platform/osgp-adapter-domain-core</module>
-    <module>osgp/platform/osgp-adapter-domain-microgrids</module>
-    <module>osgp/platform/osgp-adapter-domain-distributionautomation</module>
-    <module>osgp/platform/osgp-adapter-domain-publiclighting</module>
-    <module>osgp/platform/osgp-adapter-domain-tariffswitching</module>
-    <module>osgp/platform/osgp-adapter-domain-smartmetering</module>
-
-    <module>osgp/platform/osgp-adapter-ws-shared</module>
-    <module>osgp/platform/osgp-adapter-ws-shared-db</module>
-
-    <module>osgp/platform/osgp-adapter-ws-core</module>
-    <module>osgp/platform/osgp-adapter-ws-admin</module>
-    <module>osgp/platform/osgp-adapter-ws-publiclighting</module>
-    <module>osgp/platform/osgp-adapter-ws-tariffswitching</module>
-    <module>osgp/platform/osgp-adapter-ws-smartmetering</module>
-    <module>osgp/platform/osgp-adapter-ws-microgrids</module>
-    <module>osgp/platform/osgp-adapter-ws-distributionautomation</module>
-
-    <module>osgp/platform/osgp-adapter-kafka-distributionautomation</module>
-    <module>osgp/platform/osgp-adapter-kafka-logging</module>
-
-    <module>osgp/platform/osgp-core</module>
-    <module>osgp/platform/osgp-logging</module>
-
-    <module>osgp/platform/osgp-secret-management</module>
-
-    <!-- integration tests -->
-    <module>integration-tests/parent-integration-tests</module>
-    <module>integration-tests/cucumber-tests-core</module>
-    <module>integration-tests/cucumber-tests-execution</module>
-    <module>integration-tests/cucumber-tests-platform</module>
-    <module>integration-tests/cucumber-tests-platform-common</module>
-    <module>integration-tests/cucumber-tests-platform-distributionautomation</module>
-    <module>integration-tests/cucumber-tests-platform-microgrids</module>
-    <module>integration-tests/cucumber-tests-platform-publiclighting</module>
-    <module>integration-tests/cucumber-tests-platform-smartmetering</module>
-    <module>integration-tests/cucumber-tests-protocol-iec60870</module>
-
-    <!-- demo client application -->
-    <module>public-lighting-demo-app/parent-os-webapps</module>
```

Comment:
Are the demo app modules completely removed?

Reply:
Missed that one, put it back